src/common/com/intellij/plugins/haxe/ide/info/HaxeParameterInfoHandler.java

```diff
@@ -40,102 +48,227 @@ public boolean couldShowInLookup() {
   }
 
   @Override
-  public Object[] getParametersForLookup(LookupElement item, ParameterInfoContext context) {
-    final Object o = item.getObject();
-    if (o instanceof PsiElement) {
-      final PsiElement element = (PsiElement)o;
+  public Object[] getParametersForLookup(@NotNull LookupElement item, ParameterInfoContext context) {
```

Comment:
Readability is in the eye of the beholder, I guess.  In very local contexts, I like the smaller variable names - they're easier to pick out visually in the code.  I do not use very short names in larger functions where the declaration of the variable is removed from its use.

(Of course, me being a lazy typist has nothing to do with my preferences. XD  )